targets/python/templates/API.py.ejs

```diff
@@ -0,0 +1,34 @@
+import PlayFabErrors
+import PlayFabHTTP
+import json
+import PlayFabSettings
+
+<%- generateApiSummary("", api, "description")%>
+<% for(var i in api.calls) { var apiCall = api.calls[i];
+%><%- generateApiSummary("", apiCall, "summary")
+%><%- getDeprecationAttribute("", apiCall)
+%>def <%- apiCall.name %>(request, customData, extraHeaders, postProcessingFunc):
+<%- getRequestActions("    ", apiCall) %>
+    playfabResult = PlayFabHTTP.DoPost("<%- apiCall.url %>", request, <%- getAuthParams(apiCall) %>, extraHeaders)
+    if type(playfabResult) is PlayFabErrors.PlayFabError:
+        error = playfabResult
+        postProcessingFunc(None, error)
+        return error
+
+    returnResult = playfabResult
+<%- getResultActions("    ", apiCall, api) %>
+    postProcessingFunc(returnResult, None)
+
+<% } %><% if (hasClientOptions) {
+%># Determine if the client session ticket is set, without actually making it public
+def IsClientLoggedIn():
+    return not (ClientSessionTicket == "" or ClientSessionTicket == None)
+
+def MultiStepClientLogin(needsAttribution):
```

Comment:
This should accept an entire SettingsForUser object, and not just the one boolean from that object.
Very few of the SDKs do this in the "latest" way, since it's recently changed.